main.py

```diff
@@ -1,7 +1,26 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
-from django.db.models import QuerySet
 
+def main() -> None:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
 
-def main() -> QuerySet:
-    pass
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
```

Comment:
The first name 'Kianu' and last name 'Reaves' seem to be typos. They are corrected later in the code, but it's better to use the correct spellings 'Keanu' and 'Reeves' from the beginning.